cantabular/contract.go

```diff
@@ -39,13 +39,36 @@ type StaticDatasetQuery struct {
 	Dataset StaticDataset `json:"dataset" graphql:"dataset(name: $name)"`
 }
 
+// GetDimensionsByNameRequest holds the request variables required from the
+// caller for making a request to obtain dimensions (Cantabular variables) by name
+// POST [cantabular-ext]/graphql
+type GetDimensionsByNameRequest struct {
```

Comment:
I get that we have a convention of one contract type per client method, but this struct doesn't even express an opinion about the variable being a name and consequently is identical to `GetDimensionOptionsRequest`. Is that ok?

Reply:
It makes sense, I will replace 'variables' with 'names' in this struct (from the caller's point of view, it doesn't matter if it's encoded as 'variables' in the query, it is just a list of names for the dimensions that we want to retrieve).